admin/app/jobs/RefreshFrontsJob.scala

```diff
@@ -63,8 +54,7 @@ object RefreshFrontsJob extends Logging {
   def runAll(): Option[Seq[Unit]] = {
     Configuration.aws.frontPressSns.map(Function.const {
       log.info("Putting press jobs on Facia Cron (MANUAL REQUEST)")
-
-      for {update <- getAllCronUpdates.getOrElse(Nil)}
+      for {update <- getAllCronUpdates}
```

Comment:
if it's only one line seems neater to use () instead of {} - I think the {} only changes the behaviour of multi line to not need `;` - but that's only a minor comment 😛